src/Microsoft.TemplateEngine.Edge/LocalizableStrings.resx

```diff
@@ -351,6 +351,10 @@ Details: {1}.</value>
     <value>Failed to create template.
 Details: {0}</value>
   </data>
+  <data name="TemplateCreator_TemplateCreationResult_Error_NoDefaultName" xml:space="preserve">
+    <value>Failed to create template due naming issues</value>
+    <comment> Template does not contain a DefaultName that can be used when --name is not passed </comment>
```

Comment:
```suggestion
   <comment>Template configuration requires the name to specified but does not configure a default name that can be used when name is not specified. Specify the name for the template when instantiating the template.</comment>
```

Please avoid `--name` syntax, as it is only applicable for CLI. Just "name" should be good enough.

Reply:
Ignore this suggestion please - my fault, the error message should be in `value` field. `comment` is just for translator. See https://github.com/dotnet/templating/pull/5744#discussion_r1058854329 instead.